getting-started/installation-apt-debian.md

```diff
@@ -33,30 +33,23 @@ sudo sh -c "echo 'deb https://packagecloud.io/timescale/timescaledb/debian/ `lsb
 wget --quiet -O - https://packagecloud.io/timescale/timescaledb/gpgkey | sudo apt-key add -
 sudo apt-get update
 
-# To install for PG 10.2+
-sudo apt-get install timescaledb-postgresql-10
-# To install for PG 9.6.3+
-sudo apt-get install timescaledb-postgresql-9.6
-
-# PG 11 support is currently in BETA. To install for PG 11.0+
-sudo apt-get install timescaledb-postgresql-11
+# Now install appropriate package for PG version
+sudo apt-get install timescaledb-postgresql-:pg-version:
 ```
 
 #### Update `postgresql.conf`
 
 >:TIP: The usual location of `postgres.conf`
-is `/etc/postgresql/9.6/main/postgresql.conf` for 9.6 and
-`/etc/postgresql/10/main/postgresql.conf` for 10, but this may vary
+is `/etc/postgresql/:pg-version:/main/postgresql.conf`, but this may vary
 depending on your setup. If you are unsure where your `postgresql.conf` file
-is located, you can query PostgreSQL through the psql interface using `SHOW config_file;`.
-Please note that you must have created a `postgres` superuser so that you can access the psql
-interface.
+is located, you can query PostgreSQL through the `psql` interface using
+`SHOW config_file;`. Please note that you must have created a `postgres`
```

Comment:
The postgres user is created during initdb unless you specify to create a different superuser but its the default. You might have to adjust pg_hba.conf to use psql as postgres depending on how it is configured.

Reply:
So some of these platforms run `initdb` for you, and others do not. I believe Debian/Ubuntu do, but cannot remember 100% for Homebrew and RPM. It can leave the user in a bit of lurch, so I thought I'd leave it. I can also remove the sentence as I don't think we've had anyone ask about it in a long time.